hack/install-dependencies.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export REPO_ROOT_DIR=${REPO_ROOT_DIR:-$(git rev-parse --show-toplevel)}
+
+export EVENTING_CONFIG=${EVENTING_CONFIG:-"./third_party/eventing-latest/"}
+export EVENTING_KAFKA_CONFIG=${EVENTING_KAFKA_CONFIG:-"./third_party/eventing-kafka-broker-latest/"}
+export ISTIO_CONFIG_DIR=${ISTIO_CONFIG_DIR:-"./third_party/istio"}
+export EVENTING_ISTIO_RESOURCES_CONFIG=${EVENTING_ISTIO_RESOURCES_CONFIG:-"./third_party/istio-resources/"}
+
+export PATH="third_party/istio/bin:$PATH"
+
+kubectl create namespace knative-eventing --dry-run=client -oyaml | kubectl apply -f -
+kubectl label namespace knative-eventing istio-injection=enabled
+
+echo "Installing Eventing from ${EVENTING_CONFIG}"
+kubectl apply -Rf "${EVENTING_CONFIG}"
+
+kubectl patch deployment \
+  eventing-webhook \
+  --type merge \
+  -n knative-eventing \
+  --patch-file "${REPO_ROOT_DIR}/hack/eventing-injection-disabled.yaml"
+
+kubectl patch deployment \
+  imc-controller \
+  --type merge \
+  -n knative-eventing \
+  --patch-file "${REPO_ROOT_DIR}/hack/eventing-injection-disabled.yaml"
+
+echo "Installing Istio"
+istioctl version
```

Comment:
any specific version we want to call out as a prequisit to this?

Reply:
No, to install the version we will use you can run `hack/update-istio.sh